api/externals/controller/teacher_controller.go

```diff
@@ -81,3 +84,17 @@ func (t *teacherController) DestroyTeacher(c echo.Context) error {
 	}
 	return c.String(http.StatusOK, "Destroy Teacher")
 }
+
+// DestroyMultiTeachers
+func (t *teacherController) DestroyMultiTeachers(c echo.Context) error {
+	DestroyTeacher := new(domain.DestroyTeacherIDs)
```

Comment:
```suggestion
	DestroyTeacherIds := new(domain.DestroyTeacherIDs)
```
intの配列であるため、Idsの方がいいかと思います